src/app_engine/gcm_config_DO_NOT_CHECK_IN.json

```diff
@@ -0,0 +1,3 @@
+{
+  "GCM_API_KEY": ""
```

Comment:
can't git ignore a file that's already tracked (ie committed). It'll still show up if you edit it.